src/server/transaction.h

```diff
@@ -270,7 +270,7 @@ class Transaction {
   struct Multi {
     absl::flat_hash_map<std::string_view, LockCnt> locks;
     std::vector<std::string_view> keys;
-
+    absl::InlinedVector<bool, 4> shard_journal_write;
```

Comment:
explain what's the meaning of this field.

Reply:
We should probably use an atomic bitset in the future